host-configs/dane-toss_4_x86_64_ib-clang@16.0.6.cmake

```diff
@@ -0,0 +1,141 @@
+
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH "ON" CACHE STRING "")
+
+set(CMAKE_BUILD_RPATH "/usr/WS2/smithdev/libs/serac/toss_4_x86_64_ib/2025_04_29_14_48_21/clang-16.0.6/serac-develop-3ijr5ppzp6hmi6gyzidig4jly23222vb/lib;/usr/WS2/smithdev/libs/serac/toss_4_x86_64_ib/2025_04_29_14_48_21/clang-16.0.6/serac-develop-3ijr5ppzp6hmi6gyzidig4jly23222vb/lib64;;/usr/tce/packages/gcc/gcc-10.3.1/lib/gcc/x86_64-redhat-linux/10" CACHE STRING "")
+
+set(CMAKE_INSTALL_RPATH "/usr/WS2/smithdev/libs/serac/toss_4_x86_64_ib/2025_04_29_14_48_21/clang-16.0.6/serac-develop-3ijr5ppzp6hmi6gyzidig4jly23222vb/lib;/usr/WS2/smithdev/libs/serac/toss_4_x86_64_ib/2025_04_29_14_48_21/clang-16.0.6/serac-develop-3ijr5ppzp6hmi6gyzidig4jly23222vb/lib64;;/usr/tce/packages/gcc/gcc-10.3.1/lib/gcc/x86_64-redhat-linux/10" CACHE STRING "")
```

Comment:
i realize the setup for enzyme isn't as automated, but it would be nice in the future to create some sort of `enzyme` spack variant and specify that in the host config name, like what we do with cuda/ hip builds here https://github.com/LLNL/serac/blob/develop/scripts/spack/packages/serac/package.py#L348

Reply:
I'm a bit hesitant, since we'd eventually like Enzyme on all supported platforms/compilers, but I don't know how feasible that is.  Maybe we wait on this change for now until we understand more what platforms/compilers are supported and automate the name change in build scripts?

Reply:
that makes sense. we can leave it as is then.